commit 914569047c2b3a086fee17081c0460e8f6efb1d4
Author: Heiko Stuebner <heiko@sntech.de>
Date:   Thu Jul 16 20:02:15 2009 +0200

    exclude tests as they don't compile currently

diff --git a/Makefile.am b/Makefile.am
index 74e4f3b..cb17d49 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,7 +4,6 @@ NULL =
 
 SUBDIRS = \
   fsoframework \
-  tests \
   $(NULL)
 
 pkgconfigdir = $(libdir)/pkgconfig
diff --git a/Makefile.in b/Makefile.in
index 24d7ee9..004be89 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -214,7 +214,6 @@ TEST_PROGS =
 NULL = 
 SUBDIRS = \
   fsoframework \
-  tests \
   $(NULL)
 
 pkgconfigdir = $(libdir)/pkgconfig
